### PR TITLE
Doc: Minor adjustment of `socket.create_server` docs

### PR DESCRIPTION
**Pull request by pavel-lexyr**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@pavel-lexyr

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/socket.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
 socket.setdefaulttimeout() -- set the default timeout value
 create_connection() -- connects to an address, with an optional timeout and
                        optional source address.
+create_server() -- creates a TCP server socket and binds it to a specified
```
Comment (nanjekyejoannah) [inline Lib/socket.py]: I would just say `TCP socket`, am no English native speaker though.

Comment (pavel-lexyr) [reply Lib/socket.py]: Explicit is better than implicit - [PEP 20](https://www.python.org/dev/peps/pep-0020/) :)

File: Lib/socket.py [hunk 2]
```diff
@@ -28,6 +28,8 @@
...
+                   address.
```
Comment (erlend-aasland) [inline Lib/socket.py]: I agree with Joannah. Also note that all lines in this section use the imperative mood.

```suggestion
create_server() -- create a TCP socket and bind it to a specified address.
```

File: Doc/library/socket.rst [hunk 3]
```diff
@@ -661,7 +661,7 @@ The following functions all create :ref:`socket objects <socket-objects>`.
 .. function:: create_server(address, *, family=AF_INET, backlog=None, reuse_port=False, dualstack_ipv6=False)
 
    Convenience function which creates a TCP socket bound to *address* (a 2-tuple
-   ``(host, port)``) and return the socket object.
+   ``(host, port)``) and returns the socket object.
```
Comment (erlend-aasland) [inline Doc/library/socket.rst:664]: Note: Other methods in this file are documented using the imperative mood. Rewording this whole section (this method) is out of scope of this PR, though.

File: Lib/socket.py (hunk 1 above)
Comment (erlend-aasland) [reply Lib/socket.py]: I applied Joannah's suggestion.

**Pull request merged**